src/dash-table/derived/edges/data.ts

```diff
@@ -30,17 +34,84 @@ export default memoizeOneFactory((
     traverse2(
         data,
         columns,
-        (datum, column, i, j) => edges.setEdges(i, j, getDataCellEdges(datum, i + offset.rows, column)(styles))
+        (datum, column, i, j) => edges.setEdges(i, j, getDataCellEdges(
+            datum,
+            i + offset.rows,
+            column,
+            false,
+            false
+        )(styles))
     );
 
-    if (active_cell) {
-        edges.setEdges(active_cell.row - offset.rows, active_cell.column - offset.columns, {
-            borderBottom: [Environment.activeEdge, Infinity],
-            borderLeft: [Environment.activeEdge, Infinity],
-            borderRight: [Environment.activeEdge, Infinity],
-            borderTop: [Environment.activeEdge, Infinity]
-        });
+    return edges;
+}
+
+const getter = (
+    baseline: EdgesMatrices | undefined,
+    columns: Columns,
+    styles: IConvertedStyle[],
+    data: Data,
+    offset: IViewportOffset,
+    activeCell: ICellCoordinates | undefined,
+    selectedCells: SelectedCells
+) => {
+    if (!baseline) {
+        return baseline;
     }
 
+    const edges = baseline.clone();
+
+    const cells = selectedCells.length ?
+        selectedCells :
+        activeCell ? [activeCell] : [];
+
+    const activeStyles = styles.filter(style => style.checksActive());
+    const inactiveStyles = styles.filter(style => !style.checksActive());
```

Comment:
https://ramdajs.com/docs/#partition

Reply:
With active/selected/default there's now three buckets of styles, partition won't work.